frontend/src/Components/Topbar.tsx

```diff
@@ -89,8 +89,15 @@ const WatchButton = observer(() => {
 });
 
 const NotificationsButton = observer((props: React.ComponentPropsWithRef<'button'>) => {
-    const {notificationsCount} = useAppState();
+    const {unreadNotificationsCount, visibleNotificationsCount} = useAppState();
     return (
-        <button {...props} disabled={notificationsCount === 0} className={notificationsCount > 0 ? styles.active : ''}><NotificationIcon /><span className={styles.label}>{notificationsCount > 0 ? notificationsCount : ''}</span></button>
+        <button {...props} disabled={visibleNotificationsCount === 0}
+                className={unreadNotificationsCount > 0 ? styles.active : ''}><NotificationIcon />
+            <span className={styles.label}>
+                {visibleNotificationsCount > 0 ?
```

Comment:
two ternary if statements are really hard to read.
unreadNotificationsCount !== visibleNotificationsCount ? `${unreadNotificationsCount}/${visibleNotificationsCount}` : (visibleNotificationsCount || '')

Reply:
That's not the same condition. I extracted the condition into explicit `if` statements. Hope it's more readable now.